### DISCUSSION TITLE
[mini event] pas d'idée (béta)

### DISCUSSION THREAD

**Discussion by sciencophage** [Crownicles/Crownicles#893] [Suggestions fermées]
Bonjour. Je suis rédacteur sur ce bot, et n'ayant pas d'idée, j'ai eut l'idée d'écrire un événement qui raconte mon absence d'idées. Mais j'oublie que vous autres chevaliers ne devez pas rédiger souvent, n'avez pas idée de ce que c'est de ne pas avoir d'idée. Et bien on pourrais avoir dans l'idée que l'absence d'idées est caractérisée par la présence vide, mais c'est là que ca trompe, par le fait qu'il s'agit de l'absence de tout. L'idéal serait que mes idéaux soient déifiés de manière à ce que la plèbe identifie ma conception de ceci. Certain diront après la lecture de ce texte que j'ai une idée fixe, mais je les ignore superbement.
Cqid, euh pardon, cqfd.

**Comment by ghost** [comment]
je ne pensais pas à une idée fixe mais à de la farine perso 🤣

**Reply by sciencophage** [reply]
euuh de la farine?

**Reply by sciencophage** [reply]
je ne comprend juste pas 🤷‍♂️

**Reply by ghost** [reply]
farine everywhere 🤣

**Reply by beaud37210** [reply]
Euuuuuh, tu es sur que tout le monde comprendra ton event

**Reply by sciencophage** [reply]
Bien sûr en plus ça crée une interaction entre les rédacteurs et le site joueurs c’est important.

**Reply by LeHoublonPensant** [reply]
> Bien sûr en plus ça crée une interaction entre les rédacteurs et le site joueurs c’est important.

Ouais, c'est un point sur lequel on n'insiste peut-être pas assez. Le jeu est plus vivant, ça fait plus amical et moins le bot "froid" (attention j'ai pas dit que DraftBot l'était hein mais ça contribue, c'est le but des rédacteurs. Et puis ça met en valeur notre travail non ? 😆)

**Comment by BastLast** [comment]
je ferme cette suggestion car je ne la comprend pas
